features/steps/header.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+from behave import given, then
+
+from docx import Document
+from docx.header import Header
+
+from helpers import test_docx
+
+
+# given ===================================================
+
+@given('a section')
+def given_a_section(context):
+    document = Document(test_docx('doc-default'))
+    context.section = document.sections[0]
```

Comment:
Hmm. Ok. Just FYI all three tests did run for me locally and all three failed at the assert step.

`doc-default` is an existing test file. I chose it because it had a single section whereas `sct-section-props` has like 3 so I thought that might muddy the waters. But I suppose it doesn't really matter. We'll just always use `sections[0]` for these tests that aren't specifically testing multi-section functionality.

It makes sense though that we want all `sct` tests to use test files that start with `sct`. I'll stick with that convention.

...

What are next steps here? Maybe take a stab at unit tests of section.header?

`class DescribeSections` in `tests.test_section` seems like the unit tests for these acceptance tests:

```
@then('I can access a section by index')
def then_I_can_access_a_section_by_index(context):
    sections = context.sections
    for idx in range(3):
        section = sections[idx]
        assert isinstance(section, Section)


@then('I can iterate over the sections')
def then_I_can_iterate_over_the_sections(context):
    sections = context.sections
    actual_count = 0
    for section in sections:
        actual_count += 1
        assert isinstance(section, Section)
    assert actual_count == 3


@then('len(sections) is 3')
def then_len_sections_is_3(context):
    sections = context.sections
    assert len(sections) == 3, (
        'expected len(sections) of 3, got %s' % len(sections)
    )
```

is that a decent template to use?

I've definitely got some reading to do about how `@pytest.fixture` works.